### ISSUE TITLE
Definition of Quadratic Voting

### ISSUE THREAD

**Issue by doctor-gonzo** [DemocracyEarth/paper#369]
I am a bit confused about the role of Quadratic Voting (QV) in the democracy earth platform 

The way DEF seems to usually speak about QV involves the number of votes you can delegate to another person (it has to a number that comes from 2^x, so 1, 2, 4, 8, 16, 32, 64, 128 etc) 

In every other instance that I have read about QV, it has to do with balancing the "tyranny by majority" effects of 1-person-1-vote and creating a mechanism by which passionate minority groups can use more than one of their votes on any one particular issue, but at progressively greater cost with each vote (for example, 1 counted vote would cost 1 vote, 2 counted votes would cost 4 votes, 3 counted votes would cost 9 votes and so on). This is the meaning employed by [Vitalik Buterin](https://medium.com/@VitalikButerin/liberation-through-radical-decentralization-22fc4bedc2ac) and [Glen Weyl](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2003531) in his book _Radical Markets_

This seems especially important if votes are able to be purchased on the open market, because it allows a dedicated minority to make themselves heard (a group of 1000 dedicated people could fairly easily amplify their voting influence by a factor of 10 if they were willing to sacrifice votes on other issues) , but does not allow tyranny by majority (a single rich person buying 1,000,000 votes on the open market could be counterbalanced by 1000 individual people casting only one vote each on a particular issue)

If DEF wants to stick with their version, I think it might be wise to change the name to something else so as not to cause confusion

**Comment by herbstephens** [comment]
Yes .. maybe we need to consider a new name other than "Quadratic Voting" .. especially because our method is more robust.  In the 'standard' QV model (where votes are squared), 20 votes would cost only 400.  In the DEF model (where each additional vote costs 2x the previous), 20 votes would cost 524,288 ... 1,311x more expensive.  

Maybe "Exponential Voting"?

**Comment by doctor-gonzo** [comment]
that makes sense to me! although I don't think it should "cost" someone more if they wish to delegate all of their votes to someone else (because they trust their judgement etc), I think it should only "cost" more (in terms of number of votes counted for the number of votes spent) at the point of actual use / if all of a person's delegated votes are used on a single issue. 

This distinction might introduce some software/tracking complications though, because the receiver of delegated votes would then have to decide _whose_ votes (out of all of the people who have delegated votes to them) rather than just the total _quantity_ they want to spend on an issue, since spending different people's votes in different quantities might be more/less effective

In other words: if a receiver of delegations was representing 100 people on an issue, it wouldn't make sense to treat that delegator as a single voter using a large quantity of their own votes, they should be "counted" (in terms of cost) as though it was those 100 people all voting on the issue themselves
